src/exporthtml.ts

```diff
@@ -126,23 +126,32 @@ function generateTranscript<T extends ReturnTypes>(messages: discord.Message[],
 
         // message content
         if(message.content) {
-            const messageContentContent = document.createElement('div');
-            messageContentContent.classList.add('chatlog__content');
-
-            const messageContentContentMarkdown = document.createElement('div');
-            messageContentContentMarkdown.classList.add('markdown');
-
-            const messageContentContentMarkdownSpan = document.createElement('span');
-            messageContentContentMarkdownSpan.classList.add('preserve-whitespace');
-            messageContentContentMarkdownSpan.innerHTML = formatContent(
-                message.content,
-                channel,
-                message.webhookId !== null
-            );
-
-            messageContentContentMarkdown.appendChild(messageContentContentMarkdownSpan);
-            messageContentContent.appendChild(messageContentContentMarkdown);
-            messageContent.appendChild(messageContentContent);
+            if (message.content.startsWith('http://') || message.content.startsWith('https://')) {
```

Comment:
I would use a URL regex instead of checking to see if the content starts with http(s)://

Reply:
On it.

Reply:
Fixed.